### DISCUSSION TITLE
Congratulations to Serentiy Team!

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#6339] [General]
Today, I have seen that the "Serenity" presentation is shown in the Visual Studio Toolbox. Which is a good achievement for Serenity. Though I think it is too late. It should be featured at least 5 years ago.

https://docs.microsoft.com/en-us/shows/visual-studio-toolbox/building-web-apps-with-the-serenity-platform

Congratulation to the master serenity author @volkanceylan.

In the presentation I have missed @VictorTomaili 

Congratulation to @furkanevran.

**Comment by ra-bedo11** [comment]
I saw it, but I had a comment. We want to learn it more, but the educational content is very few and poor. What is the solution?
We want more educational videos and more projects

**Reply by dfaruque** [reply]
Yes we need such contribution to the serenity project.
A youtube channel should be created by someone. since it is a open source project, VolkanCeylan alone cannot maintain all of them (tutorial etc.)
I was thinking about creating video content but my presentation quality is not that much good.

**Reply by edwardch** [reply]
It is probably a good idea to determine the type of content that people are looking for before we start. Then we can prioritise content. 

Eg. 
- installation
- design
- hosting/CI
- integrations
- best practice

From that point, we (the community) can at least start contributing in different ways.

- Wiki entries
  - Very easy to use, and kept together with this repo
- Documentation
  - Similar to wiki - easy for community to contribute, submit PR's, etc
- Videos 
  - Can be difficult with language barriers and people's presentation & production skills
- Blog posts
  - Guest/community contributors can submit articles to Volkan & team to publish
  - Or individuals can blog, we just need a way to show all community members the links
- GitHub discussions
  - Currently being used, but can be challenging to organise and search for common issues, already see the same things pop up a few times
- Funding
  - If we individuals can't contribute with the above items, we could fund someone to write wiki, docs, blog entries, etc...

Should this be it's own discussion to see what content people are looking for?

**Reply by ra-bedo11** [reply]
> I was thinking about creating video content but my presentation quality is not that much good.


We support you

Start

**Reply by ra-bedo11** [reply]
> It is probably a good idea to determine the type of content that people are looking for before we start. Then we can prioritise content.

good

**Reply by ra-bedo11** [reply]
There is also a problem with the version of the core regarding the Arabic language, and the directions from right to left in the text width in the forms (width) do not work

and also a problem with exporting a pdf file that contains Arabic letters

**Comment by axelcasauran** [comment]
congratulations!!! cheers!!

**Comment by kilroyFR** [comment]
Congratulations Volkan and Team Serenity, it's well deserved, and still being the best framework. No competitors.
